CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
   Now you can compare everything that is `AsRef<str>` (such as `String` and `str`
   from the standard library) to uefi strings. Please head to the documentation of
   `EqStrUntilNul` to find out limitations and further information.
+- **Breaking**: renamed feature `ignore-logger-errors` to `panic-on-logger-errors` so that it is
+  additive. It is now a default feature.
```

Comment:
nit: I think this belongs in the `Changed` section rather than `Added`

Reply:
Good catch!